### ISSUE TITLE
certain characters may lead to the user seeing "undefined"

### ISSUE THREAD

**Issue by Pomax** [mozilla/codemoji#202]
Given [how the code in ui.js forms the blue boxes with white letters](https://github.com/mozilla/codemoji/blob/master/assets/js/ui.js#L138-L139), it's possible for a user to have an input that gets converted to the word "undefined" rather than an SVG image, because the specific letter is one for which we don't have an SVG file lined up.

Something like:
```
if (!currentTwomoji) {
  // unknown codemoji 'letter' without corresponding SVG file
  return '';
}
return openspan + currentTwemoji + closespan
```
perhaps?

**Comment by anxolerd** [comment]
@Pomax: according to https://github.com/mozilla/codemoji/blob/master/assets/js/ui.js#L138 it will not happen that way:

 If `current` is undefined the else clause in ternary operator will be used and hence, the `toTwemoji(_emojiLetterSet['symbol'])` will be used.

However, `undefined` may still appear in text, when user uses characters which do not belong to alphabet. Please, see #203 for details and possible solution.

**Comment by ScottDowne** [comment]
This was merged in #203, it's on prod. Closing.
